tools/ci/check_stability.py

```diff
@@ -250,16 +250,13 @@ def main():
     venv.install("requests")
 
     args, wpt_args = get_parser().parse_known_args()
-    return run(venv, wpt_args, **vars(args))
+    return setup_and_run(venv, wpt_args, **vars(args))
 
 
-def run(venv, wpt_args, **kwargs):
-    global logger
-
+def setup_and_run(venv, wpt_args, **kwargs):
```

Comment:
This will break the `wpt check-stability` command since it uses `run` as the entry point.

Reply:
Renamed the other run to `stability_run` instead.